swift/apple/FirezoneKit/Sources/FirezoneKit/Stores/Store.swift

```diff
@@ -126,7 +138,7 @@ public final class Store: ObservableObject {
 
     IPCClient.subscribeToVPNStatusUpdates(session: session, handler: vpnStatusChangeHandler)
 
-    let initialStatus = IPCClient.sessionStatus(session: session)
```

Comment:
This `sessionStatus` seemed like a useless abstraction now with IPCClient as a enum class.